src/containers/audio-rewards-page/Tiles.tsx

```diff
@@ -49,41 +32,16 @@ export const Tile: React.FC<TileProps> = ({ className, children }) => {
   )
 }
 
-export const ClaimTile = ({ className }: { className?: string }) => {
-  const unclaimedAudio =
-    useSelector(getClaimableBalance) ?? stringWeiToBN('0' as StringWei)
-  const hasNoClaim = !unclaimedAudio || unclaimedAudio.isZero()
-  const dispatch = useDispatch()
-  const onClick = () => dispatch(pressClaim())
+export const BalanceTile = ({ className }: { className?: string }) => {
+  const balance = useSelector(getAccountBalance) ?? (new BN(0) as BNWei)
 
   return (
-    <Tile
-      className={cn([
-        styles.claimTile,
-        { [styles.claimCollapsed]: hasNoClaim },
-        className
-      ])}
-    >
+    <Tile className={cn([styles.balanceTile, className])}>
```

Comment:
nbd, but should we be consistent w/ how we pass in styles for classnames - I know that their api allows this though.